packages/grafana-ui/src/components/Tabs/styles.ts

```diff
@@ -7,6 +7,9 @@ export const getTabsStyle = stylesFactory((theme: GrafanaTheme) => {
   const tabBorderColor = selectThemeVariant({ dark: colors.dark9, light: colors.gray5 }, theme.type);
 
   return {
```

Comment:
Ref https://github.com/grafana/grafana/pull/21328#issuecomment-573015290 -> can we move the styles to the component files?